Apps/CocoaPods-FCM/src/View/DashboardView.swift

```diff
@@ -27,30 +27,36 @@ struct DashboardView: View {
 
     @EnvironmentObject var userManager: UserManager
     var body: some View {
-        ZStack {
-            VStack {
-                SettingsButton {
-                    subscreenShown = .settings
+        ScrollView {
+            ZStack {
+                VStack {
+                    SettingsButton {
+                        subscreenShown = .settings
+                    }
+                    .frame(maxWidth: .infinity, alignment: .trailing)
+                    .padding(.trailing, 10)
+                    Spacer()
                 }
-                .frame(maxWidth: .infinity, alignment: .trailing)
-                .padding(.trailing, 10)
-                Spacer()
-            }
-            .sheet(isPresented: .constant(subscreenShown == .settings), onDismiss: { subscreenShown = nil }) {
-                SettingsView {
-                    subscreenShown = nil
+                .sheet(isPresented: .constant(subscreenShown == .settings), onDismiss: { subscreenShown = nil }) {
+                    SettingsView {
+                        subscreenShown = nil
+                    }
                 }
-            }
 
-            ScrollView {
                 VStack(spacing: 15) {
                     if let loggedInUserEmail = userManager.email {
                         Text(loggedInUserEmail)
                     }
                     Text("What would you like to test?")
                     Group {
-                        InlineMessage(elementId: "dashboard-announcement", onActionClick: { _, _, _ in
-                            print("Custom callback received")
+                        InlineMessage(elementId: "dashboard-announcement", onActionClick: { message, actionValue, actionName in
```

Comment:
to test `onActionClick` callback being set, track an event. I copy/pasted this code from UIKit app in the on action click delegate function.